src/GraphQLParser/ParserOptions.cs

```diff
@@ -10,5 +10,11 @@ public struct ParserOptions
         /// By default, all comments are ignored.
         /// </summary>
         public IgnoreOptions Ignore { get; set; }
+
+        /// <summary>
+        /// Maximum recursion depth during parsing.
+        /// Defaults to 64 if not set.
+        /// </summary>
+        public int? MaxDepth { get; set; }
```

Comment:
Here I have a nullable `int`.  For System.Text.Json, it's just an `int`, and the docs say that a value of 0 defaults to 64.  Should we do that instead?

Reply:
From here https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonreaderoptions.maxdepth?view=netcore-3.1 ?

Reply:
I would use nullable.